.pre-commit-config.yaml

```diff
@@ -406,8 +412,9 @@ repos:
         name: Run BATS bash tests for changed bash files
         language: system
         entry: "./scripts/ci/pre_commit/pre_commit_bat_tests.sh"
-        files: ^breeze$|^breeze-complete$|\.sh$|\.bash$|.bats$
-        pass_filenames: false
+        files: ^breeze$|^breeze-complete$|\.sh$|\.bash$|\.bats$
+        exclude: ^tests/bats/in_container/.*bats$
+        pass_filenames: true
```

Comment:
See the general comment - I love that we can run the scripts individually and pass .bat files as parameters, but in pre-commit we should set pass-filenames to "false" because we always want to run all .bat tests if any of the relevant files change.